### PR TITLE
Markdown snapshots

### LINKED ISSUE #148: Improve test snapshot format

### PR DESCRIPTION
**Pull request by sebmck**
This PR updates test snapshot format to a subset of markdown. This allows for much nicer review and diffs. See "files changed" for a preview of the format.

This is separate to ava, which has a markdown file AND a binary file that contains the original values. In Rome's case we use the markdown as the source of truth.

Closes #148

### CODE REVIEW

**Comment by sebmck** [comment]
Credit to @jamiebuilds for the idea!

**Comment by jamiebuilds** [comment]
Credit forwarded to AVA

File: packages/@romejs/codec-js-manifest/__rtests__/__rsnapshots__/dependencies.snap.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# `dependencies.ts`
+
```
Comment (jamiebuilds) [inline packages/@romejs/codec-js-manifest/__rtests__/__rsnapshots__/dependencies.snap.md]: I kinda want a message about the file being auto-generated in here, and potentially a link to docs on how to update snapshots.

Comment (sebmck) [reply packages/@romejs/codec-js-manifest/__rtests__/__rsnapshots__/dependencies.snap.md]: What about just:

```
DO NOT MODIFY. This file has been autogenerated. Run `rome test TEST_NAME --update-snapshots`to update.
```

**Pull request merged**